docs/pages/api-docs/xgrid.md

```diff
@@ -0,0 +1,113 @@
+# XGrid API
+
+<p class="description">The API documentation of the XGrid React component. Learn more about the props and the CSS customization points.</p>
+
+## Import
+
+```js
+import {XGrid} from '@material-ui/x-grid';
+```
+
+## Props
+
+
+| Name | Type | Default | Description |
+|:-----|:-----|:--------|:------------|
+| <span class="prop-name required">rows<abbr title="required">*</abbr></span> | <span class="prop-type">RowsProp</span> |  | Set of rows of type [[RowsProp]]. |
+| <span class="prop-name required">columns<abbr title="required">*</abbr></span> | <span class="prop-type">Columns</span> |   | Set of columns of type [[Columns]]. |
+| <span class="prop-name">columnTypes</span> | <span class="prop-type">ColumnTypesRecord</span> |   | Extend native column types with your new column types. |
```

Comment:
Until #242 is resolved

```suggestion
```

Reply:
Why do you want to remove that?
If you create a column type like price, it's easier to use with this prop.
It's used internally for string number, boolean, checkbox column...